portfolio/src/main/webapp/style.css

```diff
@@ -82,6 +82,9 @@ section{
     margin-bottom: 20px;
     line-height: 28px;
 }
+.upload {
+    margin: 10px 50px 25px;
```

Comment:
You may also want to add a height an width for the image. The one I uploaded was too large that I had to use a scrollbar to view it,